app/models/setting.rb

```diff
@@ -2,10 +2,28 @@
 class Setting < ApplicationRecord
   validates :name, presence: true
 
+  SOUNDS_LIKE_YES = %w[ yes y true on ].freeze
+  private_constant :SOUNDS_LIKE_YES
+
+  # Instance methods
+
+  # Return true if a setting has a yes-ish value
+  def yes?
+    return true if value.downcase == I18n.t( 'yes' ).downcase
+    return true if SOUNDS_LIKE_YES.include? value.downcase
```

Comment:
Consider moving the normalization to a `before_validate` or `before_save` callback so the db stores truthy/yessy values consistently as a single value, and the normalization only needs to happen once at create/update time, and not on reads.